src/tools/genpolicy/src/daemon_set.rs

```diff
@@ -120,7 +120,10 @@ impl yaml::K8sResource for DaemonSet {
     }
 
     fn get_annotations(&self) -> &Option<BTreeMap<String, String>> {
```

Comment:
<not_rust_expert>Not related with this change, but is there a way for the `K8sResource` trait to implement a default `get_annotations()` so that it doesn't need to be define on every and each implementer?</not_rust_expert>

Reply:
Yes - I realized that too recently, but planning to address all of the these in a separate PR. Please create a github issue as a reminder if you'd like.

I am not a rust expert either and earlier I was looking for a while to inherit across 2 or more *structs* - similar to C++ - and I couldn't find that. But, more recently I've learned that a default implementation can be provided in the *trait* instead of a struct.